### PR TITLE
Workshop improvements

### PR DESCRIPTION
**Pull request by cticenhour**
This PR is a grab bag of items for improving the workshop. I plan to deal with item 4 in #21242 related to `PackedColumn` as well - either by moving the interpolation until later in the slides, or getting rid of it (as we don't seem to use it....only single values).

Needed new sections:
- [ ] how to set up VSCode
- [ ] how to use the website
- [x] Add a preconditioning page
- [ ] Add a "prowess" page with some of the most successful simulations (Cardinal comes to mind)
- [ ] Add a troubleshooting non-converged page (based off of the NEW failed_solves.md (see https://github.com/idaholab/moose/pull/24293). (@GiudGiud on it)

Major rework needed
- [x] Decomplexify the Material model. Level of complexity should not jump like that (@cticenhour working on it) (see #24604)

Issues to poach ideas and address feedback from:
https://github.com/idaholab/moose/issues/13617
https://github.com/idaholab/moose/issues/21242
#21745 
#23234
Related but nothing more to do afaict (some of these should be closed after this PR):
https://github.com/idaholab/moose/issues/5307
https://github.com/idaholab/moose/issues/18968
https://github.com/idaholab/moose/issues/13817
https://github.com/idaholab/moose/issues/21174
https://github.com/idaholab/moose/issues/21192

### CODE REVIEW

**Comment by GiudGiud** [comment]
I ll use your PR as well

File: tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_introduction.md [hunk 1]
```diff
@@ -37,13 +37,6 @@ Multi-physics Object Oriented Simulation Environment
 
 - 30M tests per week
 
-
-!---
-
-## MOOSE Team
-
-!media darcy_thermo_mech/moose_team.png style=width:85%;margin-left:auto;margin-right:auto;display:block;
-
```
Comment (cticenhour) [inline tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_introduction.md:46]: I've updated this figure to include all current and past members, so this slide should be left in.

Comment (cticenhour) [reply tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_introduction.md:46]: Resolved offline.

File: tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md [hunk 2]
```diff
@@ -35,9 +35,22 @@ first reactor to generate usable amounts of electricity: Experimental Breeder Re
...
 
 !media darcy_thermo_mech/inl_treat.png style=width:40%;margin-left:auto;margin-right:auto;display:block;
+
+!---
+
+## National Reactor Innovation Center (NRIC)
+
+NRIC is composed of two physical test beds to build prototypes of advanced nuclear reactors
+
+- DOME in the historic EBR-II facility
```
Comment (cticenhour) [inline tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md:51]:
```suggestion
- DOME (Demonstration of Microreactor Experiments) in the historic EBR-II facility
```

File: tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md [hunk 3]
```diff
@@ -35,9 +35,22 @@ first reactor to generate usable amounts of electricity: Experimental Breeder Re
...
+
+- LOTUS in the historic ZPPR Cell
```
Comment (cticenhour) [inline tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md:53]:
```suggestion
- LOTUS (Laboratory for Operation and Testing in the U.S.) in the historic Zero Power Physics Reactor (ZPPR) Cell
```

File: tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md [hunk 4]
```diff
@@ -35,9 +35,22 @@ first reactor to generate usable amounts of electricity: Experimental Breeder Re
...
+
+And an open-source [online virtual test bed](https://mooseframework.inl.gov/virtual_test_bed)
+to demonstrate advanced reactors through modeling and simulation.
```
Comment (cticenhour) [inline tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md:56]:
```suggestion
to demonstrate advanced reactors through modeling and simulation. More information about NRIC can be found at [https://nric.inl.gov](https://nric.inl.gov).
```

File: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_shape.md [hunk 5]
```diff
@@ -99,26 +99,17 @@ $\psi_8$ is associated to the "center" node, it is symmetric and $\geq 0$ on the
 
 !row!
 !col! width=33%
-!media darcy_thermo_mech/fem_quad9_phi0.png style=width:100%
-
-!style halign=center
-$\psi_0$
+!media darcy_thermo_mech/fem_quad9_phi0.png style=width:100% caption=$\psi_4$ prefix=''
```
Comment (cticenhour) [inline tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_shape.md:102]:
```suggestion
!media darcy_thermo_mech/fem_quad9_phi0.png style=width:100% caption=$\psi_0$ prefix=''
```
Copy error

File: tutorials/darcy_thermo_mech/doc/content/workshop/systems/postprocessors.md [hunk 6]
```diff
@@ -24,12 +27,22 @@ GeneralPostprocessor: operates once per execution
 ## Postprocessor Anatomy
 
 `Postprocessor` is a UserObject, so `initialize`, `execute`, `threadJoin`, and `finalize` methods
-are used for implementing the aggregation operation.
+can be defined.
+
+`initialize()`
+This is called once before every execution. Useful to reset accumulated quantities, initialize in general!
```
Comment (cticenhour) [inline tutorials/darcy_thermo_mech/doc/content/workshop/systems/postprocessors.md:33]:
```suggestion
This is called once before every execution. Useful to reset accumulated quantities, but users should initialize in general!
```

**Review by cticenhour — COMMENTED** [review]
@GiudGiud a couple suggestions based on changes I saw this morning.

File: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_shape.md (hunk 5 above)
Comment (GiudGiud) [reply tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_shape.md:102]: hehe

**Comment by GiudGiud** [comment]
Let's get a doco build so I can review if the link worked

**Comment by GiudGiud** [comment]
The link to Samplers worked. i ll add a bunch of links then

File: tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md [hunk 7]
```diff
@@ -35,9 +35,22 @@ first reactor to generate usable amounts of electricity: Experimental Breeder Re
...
 
 !media darcy_thermo_mech/inl_treat.png style=width:40%;margin-left:auto;margin-right:auto;display:block;
+
+!---
+
+## National Reactor Innovation Center (NRIC)
+
+NRIC is composed of two physical test beds to build prototypes of advanced nuclear reactors
+
+- DOME (Demonstration of Microreactor Experiments) in the historic EBR-II facility, test bed site capable of hosting operational nuclear reactor concepts that produce less than 10MW thermal power
```
Comment (cticenhour) [inline tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md:51]:
```suggestion
- DOME (Demonstration of Microreactor Experiments) in the historic EBR-II facility, a test bed site capable of hosting operational nuclear reactor concepts that produce less than 10MW thermal power.
```

File: tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md [hunk 8]
```diff
@@ -35,9 +35,22 @@ first reactor to generate usable amounts of electricity: Experimental Breeder Re
...
+
+- LOTUS (Laboratory for Operation and Testing in the U.S.) in the historic Zero Power Physics Reactor (ZPPR) Cell, test bed site capable of hosting operational nuclear reactor concepts that produce less than 500kW thermal power
```
Comment (cticenhour) [inline tutorials/darcy_thermo_mech/doc/content/workshop/intro/inl_background.md:53]:
```suggestion
- LOTUS (Laboratory for Operation and Testing in the U.S.) in the historic Zero Power Physics Reactor (ZPPR) Cell, a test bed site capable of hosting operational nuclear reactor concepts that produce less than 500kW thermal power.
```

**Review by cticenhour — COMMENTED** [review]
A few things I noticed from Abdalla's additions.

**Comment by GiudGiud** [comment]
Step 6 seems to be broken rn

I ll work on deprecation though

**Review by loganharbour — APPROVED** [review]

**Pull request merged**